src/System.Diagnostics.FileVersionInfo/src/System/Diagnostics/FileVersionInfo.cs

```diff
@@ -265,6 +265,11 @@ public string SpecialBuild
         /// </summary>
         public static FileVersionInfo GetVersionInfo(string fileName)
         {
+            // Check if fileName is a full path.
+            if (!Path.IsPathFullyQualified(fileName))
+            {
+                throw new ArgumentException("Absolute path information is required.");
```

Comment:
This error message should be localizable - `SR. ...`.

Reply:
Do you have an example of this somewhere else in the repo?

Reply:
https://github.com/dotnet/corefx/search?utf8=%E2%9C%93&q=throw+new+ArgumentException%28SR.&type=